protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -627,7 +627,12 @@ func genIdValidator(g *protogen.GeneratedFile, f *protogen.Field, varName string
 
 	// Check if any of the validations passed
 	g.P("if !isValidId {")
-	genErrorString(g, varName, string(f.Desc.Name()), errMsg)
+	if rules.GetLogOnly() {
+		printErrorString(g, varName, string(f.Desc.Name()), errMsg)
```

Comment:
Suggest to prefix the log line with `[log-only] ` so you can distinguish between requests that have actually been rejected.

Reply:
done